setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name='django_informixdb_vault',
-    version='0.0.3',
+    version='0.0.4',
```

Comment:
Annoying this is (presumably) a manual process

Reply:
It is, yeah.
I imagine with some Github tokens and some Travis magic, we could automate

Reply:
(the upload to PyPi is manual at this point too 😞 )